modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/Rails5ServerCodegen.java

```diff
@@ -281,8 +281,8 @@ public String toApiFilename(String name) {
         // replace - with _ e.g. created-at => created_at
         name = name.replaceAll("-", "_"); // FIXME: a parameter should not be assigned. Also declare the methods parameters as 'final'.
 
-        // e.g. PhoneNumberApi.rb => phone_number_api.rb
-        return underscore(name) + "_controllers";
+        // e.g. DefaultController => defaults_controller.rb
+        return underscore(name) + "s_controller";
```

Comment:
Yeah, the issue here is that we are using this for the file name and the code being executed is in Java, not Rails. Rails has the #pluralize method but there is no method in Java from a quick search.

Since this is the filename it can't be generated in Rails land.